cert-manager-configs/README.md

```diff
@@ -0,0 +1,45 @@
+# cert-manager-openshift-install
+
+## Install cert-manager and CRDs
+
+Install cert-manager following the [installation instructions for Helm v3](https://cert-manager.io/docs/installation/kubernetes/#steps):
```

Comment:
I tried a few different things here. Adding `cert-manager` to the Chart.yaml as a dependency seems undesirable because it will try to match the CRs before the CRDs have been created:

```yaml
dependencies:
- name: cert-manager
  version: "0.15.0"
  repository: "https://charts.jetstack.io"
```

Trying to install these together throws this error:

> Error: unable to build kubernetes objects from release manifest: [unable to recognize "": no matches for kind "Certificate" in version "cert-manager.io/v1alpha2", unable to recognize "": no matches for kind "ClusterIssuer" in version "cert-manager.io/v1alpha2"]

Have you seen a way with any of our other tooling to install these serially? It looks like helm will try to pool all of the different resource Kinds and install them in a specific order, but will still fail on validation.

There is also a webhook validation which requires the `cert-manager` pod to be up in order to create a ClusterIssuer so the install needs to happen in a least 2 steps.

Reply:
I've got an idea of how to do this, but let's take that as a follow-up.